dashboard_schema.json

```diff
@@ -199,7 +199,7 @@
                 "type": "string"
               },
               "resolution": {
-                "type": "integer"
+                "enum": [0.5, 1, 2, 3, 4, 5]
```

Comment:
Hmm, I had trouble understanding this and found https://github.com/prometheus/promdash/commit/24ac83eca971c99cba70a2afd15d4b81a5a1c367#diff-86fa8341d9271c16a38806d196d801e6R118.

I guess the problem is that you can't tell the slider to directly have the values `0.5`, `2`, ... and so on?

I'd still prefer if that code wouldn't actually change the original slider value though - how about simply making a local transformation to the slider value before using it to compute the `scalingFactor`?